maintenance/ebpf-and-eks.md

```diff
@@ -23,42 +24,119 @@ This how-to guide uses the following {{site.prodname}} features:
-
-3. [Save the instance off as a custom AMI](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/creating-an-ami-ebs.html) and make a note of the AMI ID
-
-#### Create a cluster with the custom AMI
-
-You must use the Calico CNI plugin rather than the AWS CNI plugin when setting up your cluster.  This is because EKS bundles an older version of Calico with EKS, which does not support eBPF mode.
-
-Using `eksctl`: start your cluster as normal following the [EKS with Calico CNI install doc](../getting-started/kubernetes/managed-public-cloud/eks#install-eks-with-calico-networking), but when creating the nodegroup, add the `--node-ami` and `--node-ami-family` settings.
-
-* `--node-ami` should be set to the AMI ID of the image built above.
-* `--node-ami-family` should be set to `Ubuntu1804` (in spite of the upgrade).
-
-For example:
-```
-eksctl create nodegroup --cluster my-calico-cluster --node-type t3.medium --node-ami auto --max-pods-per-node 100 --node-ami-family Ubuntu1804 --node-ami <AMI ID>
-```
-
-#### Adjust Calico settings for EKS
-
-You can now [enable eBPF mode](./enabling-bpf).
\ No newline at end of file
+#### Create an EKS cluster with a recent enough kernel
+
+By default, EKS uses Ubuntu 18.04 as its base image for EKS, which does not meet the kernel version requirement for 
+eBPF mode.  Below, we give a couple of options for how to get the cluster running with a suitable kernel:
+
+
+{% tabs tab-group:grp1 %}
+<label:Bottlerocket,active:true>
+<%
+
+#### Option 1: Bottlerocket
+
+The easiest way to start an EKS cluster that meets eBPF mode's requirements is to use Amazon's 
+[Bottlerocket](https://aws.amazon.com/bottlerocket/) OS, instead of the default.  Bottlerocket is a 
+container-optimised OS with an emphasis on security; it has a recent enough kernel to use eBPF mode.
+
+* To create a 2-node test cluster with a Bottlerocket node group, run the command below.  It is important to use the config-file
+  approach to creating a cluster in order to set the additional IAM permissions for Bottlerocket.
+
+  ```
+  eksctl create cluster --config-file - <<EOF
+  apiVersion: eksctl.io/v1alpha5
+  kind: ClusterConfig
+  metadata:
+    name: my-calico-cluster
+    region: us-west-2
+    version: '1.17'
+  nodeGroups:
+    - name: ng-my-calico-cluster
+      instanceType: t3.medium
+      minSize: 0
+      maxSize: 2
+      desiredCapacity: 2
+      amiFamily: Bottlerocket
+      iam:
+        attachPolicyARNs:
+        - arn:aws:iam::aws:policy/AmazonEKSWorkerNodePolicy
+        - arn:aws:iam::aws:policy/AmazonEKS_CNI_Policy
+        - arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly
+        - arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore
+  EOF
+  ```
+ 
+* Then, remove the AWS CNI DaemonSet: 
+  ```bash
+  kubectl delete daemonset -n kube-system aws-node
+  ```
+
+* Install {{site.prodname}} in VXLAN mode using the manifest:
+  ```bash
+  kubectl apply -f {{ "/manifests/calico-vxlan.yaml" | absolute_url }}
```

Comment:
This is for Calico with calico CNI. Do we want to talk about the option to use Calico with AWS CNI?